aspnet/publishing/iis.rst

```diff
@@ -1,116 +1,182 @@
+Windows Desktop Operating Systems
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Navigate to **Control Panel > Programs > Programs and Features > Turn Windows features on or off** (left side of the screen). Open the group for **Internet Information Services** and **Web Management Tools**. Check the box for **IIS Management Console**. Check the box for **World Wide Web Services**. Accept the default features for **World Wide Web Services** or customize the IIS features to suit your needs.
 
-.. include:: ./dataProtectionWarning.txt
+.. image:: pubIIS/_static/wf.png
 
-Publish from Visual Studio
------------------------------------
+Windows Server Operating Systems
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For server operating systems, use the **Add Roles and Features Wizard** via the **Manage** menu or the link in **Server Manager**. On the **Server Roles** step, check the box for **Web Server (IIS)**.
 
-1. Create an ASP.NET Core app using the ASP.NET Core **Web Application** template.
-2. In **Solution Explorer**, right-click the project and select **Publish**.
+.. image:: pubIIS/_static/rs.png
 
-.. image:: pubIIS/_static/p1.png
+On the **Role services** step, select the IIS role services you desire or accept the default role services provided.
 
-3. In the **Publish Web** dialog, on the **Profile** tab, select **File System**.
+.. image:: pubIIS/_static/role-services.png
 
-.. image:: pubIIS/_static/fs.png
+Proceed through the **Confirmation** step to enable the web server role and services.
 
-4. Enter a profile name. Click **Next**.
-5. On the **Connection** tab, you can change the publishing target path from the default *..\\..\\artifacts\\bin\\WebApp9\\Release\\Publish* folder. Click **Next**.
-6. On the **Settings** tab, you can select the configuration, target runtime version, and publish options. Click **Next**.
+.. _unlock-handlers:
 
-The **Preview** tab shows you the publish path (by default, the same directory as the ".sln" solution file).
 
-Deploy to IIS server
------------------------------------
+Install the .NET Core Windows Server Hosting Bundle
+---------------------------------------------------
 
-#. Navigate to the publish folder (*..\\..\\artifacts\\bin\\WebApp9\\Release\\Publish folder* in this sample).
-#. Copy the **approot** and **wwwroot** directories to the target IIS server. Note: MSDeploy is the recommended mechanism for deployment, but you can use Xcopy, Robocopy or another approach. For information on using `Web Deploy` see :doc:`iis-with-msdeploy`.
-#. In IIS Manager, create a new web site and set the physical path to **wwwroot**. You can click on **Browse *.80(http)** to see your deployed app in the browser.
+#. Install the `.NET Core Windows Server Hosting <https://microsoft.com/net>`__ bundle on the server. The bundle will install the .NET Core Runtime, .NET Core Library, and the ASP.NET Core Module. The module creates the reverse-proxy between IIS and the Kestrel server.
```

Comment:
> Install the `.NET Core Windows Server Hosting <https://microsoft.com/net>`__ 

Install the `.NET Core Windows Server Hosting <http://go.microsoft.com/fwlink/?LinkId=798480>`__